projects/BFDR/IJEFetalDeath.cs

```diff
@@ -3541,12 +3548,28 @@ public string MOMMNAME
         {
             get
             {
-                // TODO: Implement mapping from FHIR record location: 
-                return "";
+                string[] names = record.MotherGivenNames;
+                if (names.Length > 1)
+                {
+                    return Truncate(names[1], 50).PadRight(50, ' ');
+                }
+                return " ";
             }
             set
             {
-                // TODO: Implement mapping to FHIR record location: 
+                if (!String.IsNullOrWhiteSpace(value))
```

Comment:
this PR will need to wait until the PR with the name helper functions is merged in so it can call those helper functions here

Reply:
The fetus name PR has been merged in, rebase to get the latest changes and update this PR to use the new helper functions Update_FirstName and Update_MiddleName